<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>etaScan User Interface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scan Parameters:</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="PyDMLineEdit" name="txt_amplitude">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="precisionFromPV" stdset="0">
             <bool>false</bool>
            </property>
            <property name="channel" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="PyDMLineEdit" name="txt_channel">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="precisionFromPV" stdset="0">
             <bool>false</bool>
            </property>
            <property name="channel" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Channel:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Start:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txt_freqstop">
            <property name="text">
             <string>7.8</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Step Size:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Dwell:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txt_freqstep">
            <property name="text">
             <string>0.05</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="PyDMLineEdit" name="txt_dwell">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="precisionFromPV" stdset="0">
             <bool>false</bool>
            </property>
            <property name="precision" stdset="0">
             <number>3</number>
            </property>
            <property name="channel" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="PyDMLineEdit" name="txt_delf">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="precisionFromPV" stdset="0">
             <bool>false</bool>
            </property>
            <property name="precision" stdset="0">
             <number>3</number>
            </property>
            <property name="channel" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>DelF:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Stop:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txt_freqstart">
            <property name="text">
             <string>6.8</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Amplitude:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_launch">
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Launch Scan...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_status">
          <property name="text">
           <string>&lt;b&gt;Status:&lt;/b&gt; Idle</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QFrame" name="frm_plot1">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frm_plot2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frm_plot3">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFrame" name="frm_plot4">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
